src/app/visual/split-view-summary-action-bar/split-view-summary-action-bar-demo.component.html

```diff
@@ -0,0 +1,106 @@
+<div class="app-screenshot" id="screenshot-split-view" style="max-height: 900px; height: 900px">
```

Comment:
Code style > attributes should be on new lines.

Reply:
Fixed